test/runtests.jl

```diff
@@ -36,8 +36,12 @@ using MPSGE.JuMP.Containers
 
         solve!(m)
 
+        MPSGEResults = XLSX.readxlsx("MPSGEresults.xlsx")
+        ATable = MPSGEResults[1][:]
+        TwobyTwoScalarResults = DenseAxisArray(ATable[2:end,2:end],ATable[2:end,1], ATable[1,2:end])
```

Comment:
In general, variable names in Julia should be all lower case, with `_` to separate words. Only type names use CamelCase :)

Reply:
Done (I think/hope). Thank you for your style corrections.